spec/features/course/assessment/assessment_viewing_spec.rb

```diff
@@ -52,6 +52,8 @@
       end
 
       scenario 'I attempt the assessment from the show assessment page' do
+        # The button should still be attempt with this random submission
```

Comment:
"# This random submission does not belong to the user, so the button should still say 'Attempt'"